Use upstream_source_rev instead of the .git/index which
now corresponds to our debian packaging directory and not
the upstream GIT.
--- a/client/native/make-build-info
+++ b/client/native/make-build-info
@@ -1,7 +1,7 @@
 #! /bin/bash
 
 echo "const char *build_date = \"$(date +"%Y-%m-%d %H:%M:%S")\";"
-echo "const char *build_source_rev = \"$(< ../../.git/index)\";"
+echo "const char *build_source_rev = \"$(< ../../upstream_source_rev)\";"
 if test -f .version
 then echo "const char *build_version = \"$(< .version)\";"
 else echo "const char *build_version = \"unknown\";"
